pkg/nuctl/command/delete.go

```diff
@@ -51,13 +51,14 @@ func newDeleteCommandeer(rootCommandeer *RootCommandeer) *deleteCommandeer {
 
 type deleteFunctionCommandeer struct {
 	*deleteCommandeer
-	functionConfig functionconfig.Config
+	functionConfigs []functionconfig.Config
 }
 
 func newDeleteFunctionCommandeer(deleteCommandeer *deleteCommandeer) *deleteFunctionCommandeer {
-	commandeer := &deleteFunctionCommandeer{
+
+	var commandeer = &deleteFunctionCommandeer{
```

Comment:
Changing the notation of creating the `commandeer` variable here isn't consistent with the rest of the code, why do it?